doc/nrf/links.txt

```diff
@@ -1396,16 +1396,15 @@
 
 .. ### Source: ubuntu.com
 
+.. _`Linux - Ubuntu 24.04 LTS`: https://releases.ubuntu.com/24.04/
 .. _`Linux - Ubuntu 22.04 LTS`: https://releases.ubuntu.com/22.04/
 .. _`Linux - Ubuntu 20.04 LTS`: https://releases.ubuntu.com/20.04/
 
 .. ### Source: apple.com
 
+.. _`macOS 15`: https://www.apple.com/macos/macos-sequoia-preview/
```

Comment:
macos-sequoia was released Sept 16, so should not be necessary to add `-preview` to the url